### PR TITLE
clear and print

### PR DESCRIPTION
**Pull request by mrkno**

### CODE REVIEW

File: modules/associate.js [hunk 1]
```diff
@@ -33,10 +33,31 @@ exports.toggleAssociation = function(thread, hook, text) {
 	return true;
 };
 
+exports.printAssociations = function(api, event) {
+	var assoc = this.config[event.thread_id];
+	var message = '';
+	for (var a in assoc) {
+		message += a + ' → ' + assoc[a] + '\n';
```
Comment (fallaciousreasoning) [inline modules/associate.js]: Won't we end up with a trailing newline? Maybe {
    if (message.length > 0) { message += "/n"; }
    message += //thing
}

File: modules/associate.js [hunk 2]
```diff
@@ -33,10 +33,31 @@ exports.toggleAssociation = function(thread, hook, text) {
...
-		return;
+		return exports.match(event.body, event.thread_id, api);
+	}
+
+	if (event.body === '/associate') {
+		return exports.printAssociations(api, event);
+	}
+
+	if (event.body === '/associate clear') {
+		return exports.clear(api, event);
```
Comment (fallaciousreasoning) [inline modules/associate.js]: Also, do you think you should update the help section?

File: modules/associate.js (hunk 1 above)
Comment (mrkno) [inline modules/associate.js]: Yup, but I figured that it didn't matter cause facebook seems to just ignore the trailing whitespace. But I can change

File: modules/associate.js (hunk 2 above)
Comment (mrkno) [inline modules/associate.js]: done

**Pull request merged**